ambry-router/src/test/java/com/github/ambry/router/ChargeTesterQuotaManagerFactory.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2022 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ */
+package com.github.ambry.router;
+
+import com.codahale.metrics.MetricRegistry;
+import com.github.ambry.account.AccountService;
+import com.github.ambry.accountstats.AccountStatsStore;
+import com.github.ambry.config.QuotaConfig;
+import com.github.ambry.quota.AmbryQuotaManager;
+import com.github.ambry.quota.QuotaAction;
+import com.github.ambry.quota.QuotaEnforcer;
+import com.github.ambry.quota.QuotaException;
+import com.github.ambry.quota.QuotaManager;
+import com.github.ambry.quota.QuotaManagerFactory;
+import com.github.ambry.quota.QuotaMetrics;
+import com.github.ambry.quota.QuotaName;
+import com.github.ambry.quota.QuotaRecommendationMergePolicy;
+import com.github.ambry.quota.capacityunit.AmbryCUQuotaEnforcer;
+import com.github.ambry.rest.RestRequest;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+
+public class ChargeTesterQuotaManagerFactory implements QuotaManagerFactory {
```

Comment:
Note that this was a pre-existing class that has been moved out of NonBlockingQuotaTest.java.